src/lib/plugin.js

```diff
@@ -113,6 +113,9 @@ class FiveBellsLedger extends EventEmitter2 {
       }
     }
 
+    // Resolve ledger metadata
+    yield this.getInfo()
```

Comment:
Seems a little strange that the `getInfo` method has the side affect of setting some properties on the plugin object